### PR TITLE
SDK-3160 ios foreground changes

### PR DESCRIPTION
**Pull request by bkasman95**

### CODE REVIEW

File: _docs/_user_guide/data_and_analytics/braze_currents/event_glossary/message_engagement_events.md [hunk 1]
```diff
@@ -194,7 +194,9 @@ This event occurs when a user directly clicks on the Push notification to open t
 Push, iOS, Sends
 {% endapitags %}
 
-This event occurs if a push was sent while the iOS app was in the foreground. Whether the user sees the push when the app is in the foreground, is determined by how your developers have integrated the iOS SDK for foreground push handling detailed [here]({{site.baseurl}}/developer_guide/platform_integration_guides/ios/push_notifications/integration/#ios-10).
+Please note, this event is now deprecated on our [Swift SDK](https://github.com/braze-inc/braze-swift-sdk).
```
Comment (Bucimis) [inline _docs/_user_guide/data_and_analytics/braze_currents/event_glossary/message_engagement_events.md:197]:
```suggestion
Please note, this event is deprecated and not supported by our [Swift SDK](https://github.com/braze-inc/braze-swift-sdk).
```

**Review by Bucimis — COMMENTED** [review]
We should thoroughly audit the docs for mentions of the feature, I see another one e.g. here https://github.com/braze-inc/braze-docs/blob/1cd244a3e40452251631e1dc897033e4a616f15e/_docs/_partners/data_and_infrastructure_agility/customer_data_platform/segment/segment_for_currents.md

**Comment by bkasman95** [comment]
> We should thoroughly audit the docs for mentions of the feature, I see another one e.g. here https://github.com/braze-inc/braze-docs/blob/1cd244a3e40452251631e1dc897033e4a616f15e/_docs/_partners/data_and_infrastructure_agility/customer_data_platform/segment/segment_for_currents.md

added additional changes for Segment, Mixpanel + Amplitude

File: _docs/_user_guide/data_and_analytics/braze_currents/event_glossary/message_engagement_events.md [hunk 2]
```diff
@@ -194,7 +194,9 @@ This event occurs when a user directly clicks on the Push notification to open t
 Push, iOS, Sends
 {% endapitags %}
 
-This event occurs if a push was sent while the iOS app was in the foreground. Whether the user sees the push when the app is in the foreground, is determined by how your developers have integrated the iOS SDK for foreground push handling detailed [here]({{site.baseurl}}/developer_guide/platform_integration_guides/ios/push_notifications/integration/#ios-10).
+Please note, this event is not supported by our [Swift SDK](https://github.com/braze-inc/braze-swift-sdk).
+
+This event only occurs if a push was sent while the iOS app was in the foreground and your developers have integrated the Obj-C iOS SDK for foreground push handling detailed [here]({{site.baseurl}}/developer_guide/platform_integration_guides/ios/push_notifications/integration/#ios-10).
```
Comment (Bucimis) [inline _docs/_user_guide/data_and_analytics/braze_currents/event_glossary/message_engagement_events.md:199]: change to deprecation notice on obj-c

**Review by Bucimis — APPROVED** [review]

**Comment by bkasman95** [comment]
@Bucimis updated the docs to reflect new verbiage from refinement

**Review by Bucimis — APPROVED** [review]

**Pull request merged**